DigitalLearningSolutions.Web/Views/Shared/Components/CurrentFilters/Default.cshtml

```diff
@@ -0,0 +1,16 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Common
+@model CurrentFiltersViewModel
+
+<div id="applied-filters" @(Model.AppliedFilters.Any() ? "" : "hidden")>
+  <form method="get" asp-action="@ViewContext.RouteData.Values["action"].ToString()" asp-route-page="1">
+    <input type="hidden" name="filterBy" value="" />
+    <input type="hidden" name="searchString" value="@Model.SearchString" />
+    <div class="applied-filter-container">
+      @foreach (var filter in Model.AppliedFilters) {
+        <div class="nhsuk-tag nhsuk-tag--blue filter-tag">@filter.FilterCategory : @filter.DisplayText<span class="nhsuk-u-visually-hidden">.</span></div>
+      }
+    </div>
+
+    <input type="submit" class="nhsuk-search__submit clear-filter__submit" value="Clear filters" />

```

Comment:
I did talk to @stellake about doing something like that. I think it probably would be technically possible although I'm sure there would be some annoying behaviour to deal with e.g. each filter tag would have to become it's own single button form.

Reply:
I was imagining something like the MyAccount EditDetails page where you just have a form encapsulating all the Filter tags and the Clear All Filters button, and then each button passes a value to the same method with whichever tag has been clicked to then be cleared. But then you are more familiar with that design than I, and I've got no clue how it would work with JS, so this suggestion might be out of whack.

Reply:
Yeah either that or 1 form per button would probably work. Depends if it is something we actually want then I guess

Reply:
We agreed with HEE that this is out of scope, but we might do this in the future, so feel free to include any suggestions when you document the filtering functionality (HEE will get access to these too) 👍